popup.css

```diff
@@ -0,0 +1,292 @@
+.modal1 {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1 /* Sit on top */;
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+/* Modal Content/Box */
+.modal-content1 {
+  background-color: #fefefe;
+  margin: 15% auto; /* 15% from the top and centered */
+  padding: 20px;
+  border: 1px solid #888;
+  width: 80%; /* Could be more or less, depending on screen size */
+  border-radius: 8px;
+}
```

Comment:
- [ ] [OTPIONAL] It'd be better if you can center the popup vertically to the screen for a better UX 
- Here's how it looks:
  ![image](https://github.com/Rifat8080/Portfolio-site/assets/104968236/365cfaa7-23bc-407e-97a2-1a04be082646)